guava/src/main/java/org/jdbi/v3/guava/GuavaCollectors.java

```diff
@@ -77,26 +125,158 @@ public void set(T value) {
     public static class Factory implements CollectorFactory {
 
         private static final Map<Class<?>, Collector<?, ?, ?>> collectors =
```

Comment:
I think we could declare this variable in the upper case, because the map is immutable. The same for `mapCollectors` and `multimapCollectors`